entities/types/Display.ts

```diff
@@ -0,0 +1,31 @@
+import { isUndefined } from "lodash";
+import { isEnum, explainEnum, stringifyEnum, parseEnum } from "../../types/Enum";
+import { explainOk, explainNot, explainOr } from "../../types/explain";
+
+export enum Display {
+    FLEX = 'flex'
```

Comment:
This should have the other standard types, see https://developer.mozilla.org/en-US/docs/Web/CSS/display